tools/device_flow/src/main.rs

```diff
@@ -0,0 +1,140 @@
+use std::str::FromStr;
```

Comment:
What's the intent of this tool?

Reply:
Just a test client, it'll probably go away or go into the examples folder at some point :)